src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -167,11 +166,11 @@ public static Optional<String> getIndividualPreferredHearingChannel(String heari
     }
 
     public static Optional<String> getIndividualInterpreterLanguage(HearingOptions hearingOptions) {
```

Comment:
can you add a unit test for getIndividualInterpreterLanguage when hearing options is null